apps/internal/base/storage/storage.go

```diff
@@ -173,10 +173,10 @@ func (m *Manager) Write(authParameters authority.AuthParams, tokenResponse acces
 	environment := authParameters.AuthorityInfo.Host
 	realm := authParameters.AuthorityInfo.Tenant
 	clientID := authParameters.ClientID
+
```

Comment:
Can you please revert all the whitespace changes unrelated to the feature? They're just noise in the diff and will make history more difficult to navigate